<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>meaSure</class>
 <widget class="QWidget" name="meaSure">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>711</width>
    <height>351</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_3" rowstretch="0,2">
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBox_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string/>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="splitButton">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="source.qrc">
             <normaloff>:/config/image/split.png</normaloff>:/config/image/split.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>IP</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="EditSetIP">
           <property name="inputMask">
            <string>000.000.000.000</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="startButton">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="source.qrc">
             <normaloff>:/config/image/play_passive.png</normaloff>:/config/image/play_passive.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="stopButton">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="source.qrc">
             <normaloff>:/config/image/stop_active.png</normaloff>:/config/image/stop_active.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QGroupBox" name="groupBox_3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>基本信息</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_4" rowstretch="0,0,0,0,0,0,0,0" columnstretch="0,0,0,0,0,0,0,0,0,0,0">
         <item row="4" column="3">
          <widget class="QLineEdit" name="UcPhaseEdit"/>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>幅值</string>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QLineEdit" name="UaPhaseEdit"/>
         </item>
         <item row="4" column="2">
          <widget class="QLineEdit" name="UcVoltageEdit"/>
         </item>
         <item row="3" column="3">
          <widget class="QLineEdit" name="UbPhaseEdit"/>
         </item>
         <item row="3" column="4">
          <widget class="QLineEdit" name="UbFreqEdit"/>
         </item>
         <item row="4" column="4">
          <widget class="QLineEdit" name="UcFreqEdit"/>
         </item>
         <item row="2" column="6">
          <widget class="QLabel" name="label_6">
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 170, 0);</string>
           </property>
           <property name="text">
            <string>Ia</string>
           </property>
          </widget>
         </item>
         <item row="1" column="8">
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>相位(ch1)</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QLineEdit" name="UaVoltageEdit"/>
         </item>
         <item row="4" column="8">
          <widget class="QLineEdit" name="IcPhaseEdit_CH1"/>
         </item>
         <item row="2" column="8">
          <widget class="QLineEdit" name="IaPhaseEdit_CH1"/>
         </item>
         <item row="2" column="7">
          <widget class="QLineEdit" name="IaCurrentEdit_CH1"/>
         </item>
         <item row="4" column="6">
          <widget class="QLabel" name="label_8">
           <property name="styleSheet">
            <string notr="true">color: rgb(170, 0, 0);</string>
           </property>
           <property name="text">
            <string>Ic</string>
           </property>
          </widget>
         </item>
         <item row="3" column="7">
          <widget class="QLineEdit" name="IbCurrentEdit_CH1"/>
         </item>
         <item row="1" column="7">
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>幅值(ch1)</string>
           </property>
          </widget>
         </item>
         <item row="2" column="10">
          <widget class="QLineEdit" name="IaPhaseEdit_CH2"/>
         </item>
         <item row="2" column="9">
          <widget class="QLineEdit" name="IaCurrentEdit_CH2"/>
         </item>
         <item row="4" column="7">
          <widget class="QLineEdit" name="IcCurrentEdit_CH1"/>
         </item>
         <item row="1" column="3">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>相位</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_4">
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 255, 0);</string>
           </property>
           <property name="text">
            <string>Ub</string>
           </property>
          </widget>
         </item>
         <item row="2" column="4">
          <widget class="QLineEdit" name="UaFreqEdit"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_5">
           <property name="styleSheet">
            <string notr="true">color: rgb(170, 0, 0);</string>
           </property>
           <property name="text">
            <string>Uc</string>
           </property>
          </widget>
         </item>
         <item row="3" column="6">
          <widget class="QLabel" name="label_7">
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 255, 0);</string>
           </property>
           <property name="text">
            <string>Ib</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2" colspan="2">
          <widget class="QLineEdit" name="GPSInfoEdit"/>
         </item>
         <item row="1" column="4">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>频率</string>
           </property>
          </widget>
         </item>
         <item row="2" column="5">
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="3" column="2">
          <widget class="QLineEdit" name="UbVoltageEdit"/>
         </item>
         <item row="3" column="8">
          <widget class="QLineEdit" name="IbPhaseEdit_CH1"/>
         </item>
         <item row="4" column="5">
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_3">
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 170, 0);</string>
           </property>
           <property name="text">
            <string>Ua</string>
           </property>
          </widget>
         </item>
         <item row="3" column="5">
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="3" column="9">
          <widget class="QLineEdit" name="IbCurrentEdit_CH2"/>
         </item>
         <item row="3" column="10">
          <widget class="QLineEdit" name="IbPhaseEdit_CH2"/>
         </item>
         <item row="4" column="9">
          <widget class="QLineEdit" name="IcCurrentEdit_CH2"/>
         </item>
         <item row="4" column="10">
          <widget class="QLineEdit" name="IcPhaseEdit_CH2"/>
         </item>
         <item row="7" column="0" colspan="11">
          <widget class="QGroupBox" name="groupBox_4">
           <property name="title">
            <string>原始数据</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="0" column="0">
             <widget class="QTextEdit" name="SourceDataEdit"/>
            </item>
           </layout>
          </widget>
         </item>
         <item row="1" column="9">
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>幅值(ch2)</string>
           </property>
          </widget>
         </item>
         <item row="1" column="10">
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string>相位(ch2)</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="2">
          <widget class="QLabel" name="label_2">
           <property name="styleSheet">
            <string notr="true">image: url(:/config/image/GPS.png);</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="source.qrc"/>
 </resources>
 <connections/>
</ui>
